projects/replicated-ship/schema.json

```diff
@@ -348,6 +348,80 @@
                   "source"
                 ]
               },
+              "google_gke": {
+                "description": "An `google_gke` asset generates a terraform file that will create a Google GKE Cluster.",
```

Comment:
An -> A

Reply:
This is autogenerated though :(

Reply:
I'll make a PR into ship to fix it and then update this.